source/scss/components/top-offer-list/_top-offer-list.scss

```diff
@@ -23,3 +23,29 @@
     align-self: flex-end;
   }
 }
+
+.ratio-3-2 {  
+  display: inline-block;
+  overflow: hidden;
+  position: relative;
+  width: 100%;
+
+  ::after {
+    content: "";
+    display: block;
+    padding-top: 66.66%;
+  }
+
```

Comment:
Please talk to Denise again. If I got her right, the image should be centered centered. The cropping is OK the way, I did it and is maybe changed later.

Reply:
top to -9999 and bottom to -9999